examples/logz/subaccount.yaml

```
@@ -0,0 +1,54 @@
+apiVersion: logz.azure.upbound.io/v1beta1
+kind: SubAccount
+metadata:
+  annotations:
+    meta.upbound.io/example-id: logz/v1beta1/subaccount
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    logzMonitorIdSelector:
+      matchLabels:
+        testing.upbound.io/example-name: examplemonitor
+    user:
+    - email: user@example.com
+      firstName: Example
+      lastName: User
+      phoneNumber: "+12313803556"
+---
+apiVersion: logz.azure.upbound.io/v1beta1
+kind: Monitor
+metadata:
+  annotations:
+    meta.upbound.io/example-id: logz/v1beta1/monitor
+  labels:
+    testing.upbound.io/example-name: examplemonitor
+  name: ${Rand.RFC1123Subdomain}
+spec:
+  forProvider:
+    location: West Europe
+    plan:
+      - billingCycle: MONTHLY
+        effectiveDate: "2022-06-06T00:00:00Z"
+        usageType: COMMITTED
+    resourceGroupNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: rgsubaccount
+    user:
+      - email: user@example.com
+        firstName: Example
+        lastName: User
+        phoneNumber: "+12313803556"
+---
+apiVersion: azure.upbound.io/v1beta1
+kind: ResourceGroup
+metadata:
+  annotations:
+    meta.upbound.io/example-id: logz/v1beta1/monitor
```

could you please use the following suggestion:

```suggestion
    meta.upbound.io/example-id: logz/v1beta1/subaccount
```

---

Done